content/md/imports-exports/monitor-jobs.md

```diff
@@ -9,57 +9,91 @@ related: imports, exports, product-export-builder, discover-the-dashboard, what-
+![Jobs flash message](../img/Jobs_flash-message.png)
+1. Then, while the job is running, it will be listed in the Process Tracker amongst all the other jobs you launched. To access it, go to `Activity` and click on `Process tracker`.
+![List of jobs in process tracker](../img/Jobs_process-tracker.png)  
+Note that you can also check the status of your launched jobs on the dashboard. To access it, go to `Activity` and click on `Dashboard`.
+![List of jobs in dashboard](../img/Jobs_dashboard.png)
+1.  Finally, when the job is complete, a notification appears at the top of your screen. You can always go back to the job itself if you need more details.
+![Notification of a job complete](../img/Jobs_notification.png)
 
-# How to monitor?
-First, you are notified by a flash message when the job starts.
-![image](../img/Settings_Calculation1.png)
+::: tips
+Please note it is possible to use the `View all jobs in Process Tracker` permission in the `Roles` page to access all the jobs launched by all the users of your PIM. To find out more, check our article on [permissions](manage-the-interface-and-actions-accesses.html#rights-on-system-pages) in our help center.
+:::
 
-Then, when the job is completed, a notification appears on the top of your screen.
-![image](../img/Settings_Calculation2.png)
+# What can you find in a job detail?
 
-There is a Process Tracker page which will give you the full list of all jobs you launched. To access it, go to `Activity` and click on the `Process tracker` menu entry.
-![image](../img/Settings_Calculation3.png)
+A job detail is a report of a specific job.
+In each job detail, you'll find information about the status of your job, which may vary depending on their status: `Started`, `Completed`, `Failed` or `Stopped` with a specific color code. You will see:
+* The number of steps
+* A time estimation above each step
+* A table with the details of all the actions performed under each step
 
-The `Last operations` widget on Akeneo dashboard will also give you the status of your launched jobs.
-![image](../img/Dashboard_Calculation4.png)
+To access a job detail:
+1.  Go to `Activity`/`Process Tracker`
+1.  Search for the job you would like to know more about
+1.  Click on the job to get more details
+1.  The job detail appear
+![Job detail example](../img/Jobs_job-detail.png)
 
-::: tips
-If you want to view the jobs of all the users of your PIM (and not only yours), adding the [permission on action](manage-the-interface-and-actions-accesses.html#rights-on-system-pages) `View all jobs in Process Tracker` on your user role will grant you that privilege.
-:::
+# Searching for a job
+
+If you want to find a specific job amongst all those listed in the process tracker, you can use the following search filters:
+* By job name
+* By type
+* By user
+* By status
+![Process tracker search filters](../img/Exports_Process-Tracker-search-filters.png)
+
+# How to stop your jobs?
 
-## Imports & exports reports
```

Comment:
As you removed this title, can you make sure it's not referenced somewhere else in the help center? If it's the case, you'll need to change the link to another one in order to make it work again.

Reply:
Checked, nothing to modify here. Thanks